test/support/fixtures/pets_schema.ex

```diff
@@ -123,6 +123,21 @@ defmodule Absinthe.Fixtures.PetsSchema do
     field :string_list_field, list_of(:string)
   end
 
+  scalar :custom_scalar do
+    parse fn
+      %Absinthe.Blueprint.Input.Object{} = input ->
```

Comment:
Is this part necessary to recreate the bug?

Reply:
Probably not! I'll verify.